src/db/shares.ts

```diff
@@ -83,19 +83,29 @@ export const markShareAsEmailed = async (id: string): Promise<Share> => {
 
 export const createShare = async (
   share: ShareInsert
-): Promise<ShareWithUsername> => {
-  const res = await supabase
+): Promise<ShareWithUsername | null> => {
+  const shared = await supabase
```

Comment:
Can we go ahead and destructure the data property originally? So:

`const {data} = await supabase...`

Reply:
Done ✌